src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/TenantsApiAction.java

```diff
@@ -1,3 +1,34 @@
+/*
+ * Copyright 2015-2017 floragunn GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Since this file was added in 2019, I added the 2019 year - https://github.com/opendistro-for-elasticsearch/security/commits/master/src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/TenantsApiAction.java.

We probably need to figure this out for a lot of files that have been present for sometime in our codebase.